demo/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-relative-import-paths/no-relative-import-paths": "off"
```

Comment:
this shouldn't be here should it?

Reply:
I can pull it off onto separate PR if you desire

Reply:
I just thought you already merged it. Must have been a different PR. I've looked at to many PRs today lol.

Reply:
yeah, that was the same change but for prefect-design
https://github.com/PrefectHQ/prefect-design/pull/564